Starwar/Assets/Scripts/Player Control/Health.cs

```diff
@@ -3,7 +3,7 @@ public class Health : MonoBehaviour
 {
     public float MaxHealth;
 
-    private float currentHealth;
+    private static float currentHealth;
```

Comment:
currentHealth cannot be static. Every space ship has Health component, hence currentHealth represent the current health of this space ship